src/main/views/colorpickerView.ts

```diff
@@ -0,0 +1,133 @@
+  //       posButton: this.storage.get('buttonsPosition'),
+  //       typeButton:  this.storage.get('buttonsType'),
+  //       tools: this.storage.get('tools'),
+  //       colorfullApp: this.storage.get('colorfullApp'),
+  //       history: this.storage.get('history')
+  //     }
+
+  //     event.sender.send('init', config)
+  //   })
+
+  //   ipcMain.on('changeLastColor', (event, color) => {
+  //     this.storage.set('lastColor', color)
+  //   })
+
+  //   ipcMain.on('saveColor', (event, color) => {
+  //     let colorsbook = this.storage.get('colors', 'colorsbook')
+  //     colorsbook[Object.getOwnPropertyNames(colorsbook)[Object.values(colorsbook).length - 1]].push(color)
+  //     this.storage.set('colors', colorsbook, 'colorsbook')
+  //   })
+
+  //   ipcMain.on('changeHistory', (event, array) => {
+  //     this.storage.set('history', array)
+  //     this.util.eventEmitter.emit('updateHistory', array)
+  //   })
+
+  //   ipcMain.on('opacityActive', (event, bool) => {
+  //     opacity = bool
+  //     let size = this.window.getSize()
+  //     if (!opacity && shading) return this.window.setMinimumSize(440, 220)
+  //     if (!opacity) return this.window.setMinimumSize(440, 150)
+  //     if (size[1] < 180 && !shading) this.window.setSize(size[0], 180, true)
+  //     if (size[1] < 255 && shading) this.window.setSize(size[0], 255, true)
+  //     if (!shading) this.window.setMinimumSize(440, 180)
+  //     else this.window.setMinimumSize(440, 255)
+  //   })
+
+  //   ipcMain.on('shadingActive', (event, bool) => {
+  //     shading = bool
+  //     let size = this.window.getSize()
+  //     if (!shading && !opacity) return this.window.setMinimumSize(440, 150)
+  //     if (!shading && opacity) return this.window.setMinimumSize(440, 180)
+  //     if (size[1] < 220 && !opacity) this.window.setSize(size[0], 220, true)
+  //     if (size[1] < 255 && opacity) this.window.setSize(size[0], 255, true)
+  //     if (!opacity) this.window.setMinimumSize(440, 220)
+  //     else this.window.setMinimumSize(440, 255)
+  //   })
+
+  //   ipcMain.on('minimize-colorpicker', event => this.window.minimize())
+  //   ipcMain.on('maximize-colorpicker', event => {
+  //     if (this.window.isMaximized()) return this.window.unmaximize()
+  //     else return this.window.maximize()
+  //   })
+  //   ipcMain.on('close-colorpicker', event => this.window.close())
+  //   ipcMain.on('setOnTop', (event, bool) => this.window.setAlwaysOnTop(bool))
+
+  //   // ipcMain.on('launchPicker', event => picker.init())
+  //   // this.eventEmitter.on('launchPicker', event => picker.init())
+  //   // ipcMain.on('launchColorsbook', event => colorsbook.init())
+  //   // this.eventEmitter.on('launchColorsbook', event => colorsbook.init())
+  //   // ipcMain.on('showPreferences', event => settings.init())
+  //   // this.eventEmitter.on('showPreferences', event => settings.init())
+  // }
```

Comment:
Why don't remove this functions, instead of commenting them?

Reply:
Because I use it as a guide to rewrite in typescript for the moment. I will remove it when I don't need it